File: rewrite-java/src/main/java/org/openrewrite/java/search/HasImport.java
package org.openrewrite.java.search;

import lombok.RequiredArgsConstructor;
import org.openrewrite.Tree;
import org.openrewrite.java.JavaSourceVisitor;
import org.openrewrite.java.tree.J;

@RequiredArgsConstructor
public class HasImport extends JavaSourceVisitor<Boolean> {
    private final String clazz;

    @Override
    public Boolean defaultTo(Tree t) {
        return false;
    }

    @Override
    public Boolean visitImport(J.Import impoort) {
        return impoort.isFromType(clazz);
    }
}

File: src/main/kotlin/com/netflix/java/refactor/search/HasImport.kt
package com.netflix.java.refactor.search

import com.netflix.java.refactor.ast.visitor.AstVisitor
import com.netflix.java.refactor.ast.Cursor
import com.netflix.java.refactor.ast.Tr

class HasImport(val clazz: String): AstVisitor<Boolean>(false) {
    override fun visitImport(import: Tr.Import): Boolean = 
            import.matches(clazz)
}